docker/gaffer-kerberos/docker-compose.yaml

```diff
@@ -0,0 +1,263 @@
+# Copyright 2023 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: "3.7"
+
+networks:
+  default:
+    name: gaffer
+
+services:
+
+  kdc-server:
+    image: gchq/kdc:0.1
```

Comment:
Thinking maybe this could be called something more descriptive? Like kerberos-kdc or kdc-server or something? What do you think?

Reply:
We won't be publishing this image, but it could be `kerberos-kdc-server`, might make it easier to identify when doing `docker image ls`.